unitTests/Fake.ts

```diff
@@ -262,7 +262,7 @@ export class FakeResults {
         return { value: token + number, numberOfResults: 5 };
       }),
       parentValues: range(0, numberOfValues).map(number => {
-        return { value: 'parent' + number, numberOfResults: 5 };
+        return { value: 'parent' + number, numberOfResults: number + 250 * number };
```

Comment:
Hmm, I am curious as to why the numberOfResults needs to be changed?

Reply:
I was set to `5` for all values for the unit tests. I didn't find it useful to test 10 times if the values was to 5. 
Also, this way, we get values above `1000` to properly test the formatting (see the expected results in the unit test)